src/components/ec-btn-dropdown/ec-btn-dropdown.spec.js

```diff
@@ -51,6 +51,23 @@ describe('EcBtnDropdown', () => {
     expect(wrapper.emitted('click').length).toBe(1);
   });
 
+  it("should render an anchor tag with a 'href' prop", async () => {
```

Comment:
The description should match the assertions.

`it('should emit a click event when the CTA button is clicked')`

Reply:
corrected

Reply:
corrected